src/odemis/gui/cont/tabs.py

```diff
@@ -2877,6 +2910,8 @@ def _create_new_dir(self):
 
     def _reset_project_data(self):
         try:
+            streams = self._get_overview_view().stream_tree.streams
```

Comment:
Maybe move this line inside remove_overview_streams

Reply:
I used remove_overview_streams at one more place i.e. in line 504 of function _on_overview_visible where it takes list of invisible streams as input.

Reply:
missed that 👍